en/05_Contributing/06_Documentation.md

```diff
@@ -1,141 +1,250 @@
+* Keep documentation lines shorter than 120 characters.
+* Avoid FAQs. FAQs are not a replacement for coherent, well explained documentation. If you've done a good job
 documenting, there shouldn't be any "frequently asked questions" left.
-* "SilverStripe" should always appear without a space with both "S"s capitalised.
-* Use simple language and words. Avoid uncommon jargon and overly long words.
-* Use UK English and not US English. SilverStripe is proudly a New Zealand open source project we use the UK spelling and forms of English. The most common of these differences are -ize vs -ise, or -or vs our (eg color vs colour).
-* We use sentence case for titles so only capitalise the first letter of the first word of a title. The only exceptions to this are when using brand names (e.g. SilverStripe), acronyms (e.g. PHP) and class names (e.g. ModelAdmin).
-* Use gender neutral language throughout the document, unless referencing a specific person. Use them, they, their, instead of he and she, his or her.
-* URLs: if the end of your sentence is a URL then you don't need to use a full stop.
-* Bullet points: Sentence case your bullet points. If a bullet point is a full sentence then end with a full stop. If it is a sentence fragment or a comma separated list, full stops are not required.
-
-## Highlighted blocks
-
-There are several built-in block styles for highlighting a paragraph of text. Please use these graphical elements 
+* We use sentence case for titles. Only capitalise the first letter of the first word of a title. The only exceptions to this are when using brand names (e.g. GitHub), acronyms (e.g. PHP) and class names (e.g. ModelAdmin).
+* Use `example.com` as the domain of sample URLs and `example.org` and `example.net` when additional domains are required. All of these domains are [reserved by the IANA](https://tools.ietf.org/html/rfc2606#section-3).
+* Use appropriate markdown. The use of code blocks, emphasis, lists, and tables make technical writing easier to scan.
+* Never leave spaces at the end of a line.
+* Use an empty line to separate paragraphs.
+* Sentence case your bullet points. If a bullet point is a full sentence then end with a full stop. If it is a sentence fragment or a comma separated list, full stops are not required.
+* All code blocks must have an appropriate syntax highlighting hint. For plain text, use "text".
+* Code examples should follow our [coding conventions](./coding_conventions).
+* Use `App` when the code requires a vendor name (e.g. as the root for namespaces).
+* Do not put a `$` in front of lines of bash code examples.
+* When referencing directories, always add a trailing slash to avoid confusions with regular files
+* "Silverstripe CMS" is used to refer to the Silverstripe CMS open source project as a whole.
+* "Silverstripe" is used to refer to the company, where appropriate.
```

Comment:
```suggestion
* "Silverstripe Ltd" is used to refer to the company, where appropriate.
```

Reply:
We've been actively avoiding "Silverstripe Ltd" to refer to the company as per @maxime-rainville's instruction